app/controllers/crops_controller.rb

```diff
@@ -120,8 +135,16 @@ def create
   def update
     @crop = Crop.find(params[:id])
 
+    previous_status = @crop.approval_status
+
     respond_to do |format|
       if @crop.update(crop_params)
+        if previous_status == "pending"
+          requester = @crop.requester
+          new_status = @crop.approval_status
+          Notifier.crop_request_approved(requester, @crop).deliver! if new_status == "approved"
+          Notifier.crop_request_rejected(requester, @crop).deliver! if new_status == "rejected"
```

Comment:
Note to self: If I refactor this using `send` to get rid duplication will that be annoying and less explicit?